libs/sdk-ui-filters/src/AttributeFilter/Components/ElementsSelect/StatusBar/AttributeFilterShowFilteredElements.tsx

```diff
@@ -0,0 +1,57 @@
+// (C) 2023 GoodData Corporation
+
+import React, { ReactNode, useMemo } from "react";
+import { FormattedMessage } from "react-intl";
+import { Bubble, BubbleHoverTrigger } from "@gooddata/sdk-ui-kit";
+
+const ALIGN_POINTS = [{ align: "bl tl" }];
+const ARROW_OFFSETS = { "bl tl": [0, 8] };
+
+interface IAttributeFilterShowFilteredElementsProps {
+    isHidden: boolean;
+    attributeTitle: string;
+    onClick: () => void;
+    parentFilterTitles: string[];
+}
+
+export const AttributeFilterShowFilteredElements: React.FC<IAttributeFilterShowFilteredElementsProps> = ({
+    isHidden,
```

Comment:
This looks suspicious - should we rather hide it one level above in the component tree? (isHidden === true -> do not render this component) - then you don't need "isHidden" prop.